setup.py

```diff
@@ -1,85 +0,0 @@
-#!/usr/bin/env python3
-
-# FIXME:
-#
-# - REENABLE HG_STARTUP BENCHMARK.
-#
-# Update dependencies:
-#
-#  - python3 -m pip install --user --upgrade pip-tools
-#  - git clean -fdx  # remove all untracked files!
-#  - (pip-compile --upgrade -o requirements.txt requirements.in)
-#
-# Prepare a release:
-#
-#  - git pull --rebase
-#  - Remove untracked files/dirs: git clean -fdx
-#  - maybe update version in pyperformance/__init__.py and doc/conf.py
-#  - set release date in doc/changelog.rst
-#  - git commit -a -m "prepare release x.y"
-#  - run tests: tox --parallel auto
-#  - git push
-#  - check the CI status:
-#    https://github.com/python/pyperformance/actions
-#
-# Release a new version:
-#
-#  - git tag VERSION
-#  - git push --tags
-#  - Remove untracked files/dirs: git clean -fdx
-#  - python3 setup.py sdist bdist_wheel
-#  - twine upload dist/*
-#
-# After the release:
-#
-#  - set version to n+1: pyperformance/__init__.py and doc/conf.py
-#  - git commit -a -m "post-release"
-#  - git push
-
-# Import just to get the version
-import pyperformance
-
-VERSION = pyperformance.__version__
-
-DESCRIPTION = 'Python benchmark suite'
-CLASSIFIERS = [
-    'Development Status :: 5 - Production/Stable',
-    'Intended Audience :: Developers',
-    'License :: OSI Approved :: MIT License',
-    'Natural Language :: English',
-    'Operating System :: OS Independent',
-    'Programming Language :: Python :: 3',
-    'Programming Language :: Python',
-]
-
-
-# put most of the code inside main() to be able to import setup.py in
-# unit tests
-def main():
-    import os.path
-    from setuptools import setup, find_packages
-
-    with open('README.rst', encoding="utf8") as fp:
-        long_description = fp.read().strip()
-
-    options = {
-        'name': 'pyperformance',
-        'version': VERSION,
-        'author': 'Collin Winter and Jeffrey Yasskin',
-        'license': 'MIT license',
-        'description': DESCRIPTION,
-        'long_description': long_description,
-        'url': 'https://github.com/python/benchmarks',
-        'classifiers': CLASSIFIERS,
-        'packages': find_packages(),
-        'include_package_data': True,
```

Comment:
What handles the package data in pyproject.toml?

Reply:
https://github.com/python/pyperformance/pull/249/commits/f20a4635ee1798a8b6546af415ca8dfdfb078549

Updated! Thanks for catching!